src/base/QXmppRtpChannel.cpp

```diff
@@ -352,7 +352,7 @@ void QXmppRtpAudioChannel::datagramReceived(const QByteArray &ba)
     QXmppCodec *codec = 0;
     const quint8 packetType = packet.type();
     if (!d->incomingCodecs.contains(packetType)) {
-        foreach (const QXmppJinglePayloadType &payload, m_incomingPayloadTypes) {
+        for (const auto &payload : m_incomingPayloadTypes) {
```

Comment:
I'd much rather rewrite this with `std::find_if` to better express the intent, but ultimately it's up to you.